hack/run-e2e-test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -uo pipefail
+
+# Fetching ginkgo for running the test
+GO111MODULE=on go get -u github.com/onsi/ginkgo/ginkgo
```

Comment:
`ginkgo` is already in `go.mod`, so there is no reason to explicitly turn modules on, or use the `-u` flag. A simple `go get` will download whatever version is in the `.mod file`.